packages/tools/devtools/devtools-view/src/components/OpLatencyView.tsx

```diff
@@ -84,13 +91,27 @@ export function OpLatencyView(): React.ReactElement {
 			},
 			data: [],
 		});
-	const unsampledTelemetry = localStorage.getItem("Fluid.Telemetry.DisableSampling");
+	const [unsampledTelemetry, setUnsampledTelemetry] = React.useState(
+		(): boolean => localStorage.getItem("Fluid.Telemetry.DisableSampling") === "true",
+	);
+
 	// Render the text conditionally
-	const renderInstructions = unsampledTelemetry !== "true";
+	const renderInstructions = unsampledTelemetry !== true;
 
 	function formatTimestamp(timestamp: number): string {
 		return new Date(timestamp).toTimeString().slice(0, 8);
 	}
+
+	const toggleUnsampledTelemetry = (): void => {
```

Comment:
Once we set the value in localStorage/sessionStorage, I believe the tab needs to be refreshed for Fluid to start using the new value. Can we make the button trigger a refresh as well? And if so, we should also add a noticeabl bit to the instructions saying that the page will refresh when the users clicks.